src/ReadModel/DocumentDoesNotExistException.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\ReadModel;
+
+use Exception;
+
+final class DocumentDoesNotExistException extends Exception
```

Comment:
Can we get rid of the `Exceptions` postfix here?

Reply:
Good point, I'll remove that!